packages/system-tests/src/spectron/application.ts

```diff
@@ -111,9 +111,23 @@ export class SpectronApplication {
     );
   }
 
+  private getKeybindingPlatform(): string {
```

Comment:
I followed VSCode's implementation for this - https://github.com/Microsoft/vscode/blob/master/test/smoke/src/main.ts#L118